helm/bankapp/templates/mysqlService.yml

```
@@ -0,0 +1,15 @@
+  name: "{{ .Values.db_statefulset.name }}-headless"
+  namespace: {{ default "bankapp-namespace" .Values.namespace }}
+  labels:
+    app: {{ .Values.db_statefulset.name }}
+spec:
+  clusterIP: None
+  selector:
+    app: {{ .Values.db_statefulset.name }}
+  ports:
+  - protocol: TCP
+    port: {{ default "3306" .Values.mysql_svc.port}}
+    targetPort: {{ default "3306" .Values.mysql_svc.targetPort}}
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)